pkg/daemon/ceph/osd/volume.go

```diff
@@ -459,6 +460,19 @@ func isSafeToUseRawMode(device *DeviceOsdIDEntry, cephVersion cephver.CephVersio
 	return true
 }
 
+func isSupportOnlyRawMode(device *DeviceOsdIDEntry, storeConfig *config.StoreConfig) bool {
+	if device.DeviceInfo.Type == sys.LVMType {
+		logger.Infof("%q is not picked since LVM logical volume doesn't support `metadataDevice` and/or `osdsPerDevice` > 1", device.Config.Name)
+		return true
```

Comment:
If the method is renamed like I suggested, these conditions would be swapped.
```suggestion
		return false
```